molgenis-app/pom.xml

```diff
@@ -59,29 +60,42 @@
 			</resource>
 		</resources>
 		<plugins>
-            <!-- minify resources -->
-            <plugin>
-                <groupId>com.samaxes.maven</groupId>
-                <artifactId>minify-maven-plugin</artifactId>
-            </plugin>
+			<!-- minify resources -->
+			<plugin>
+				<groupId>com.samaxes.maven</groupId>
+				<artifactId>minify-maven-plugin</artifactId>
+			</plugin>
 			<!-- make molgenis server properties available in pom.xml -->
 			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>properties-maven-plugin</artifactId>
-				<executions>
-					<execution>
-						<phase>initialize</phase>
-						<goals>
-							<goal>read-project-properties</goal>
-						</goals>
-						<configuration>
-							<files>
-								<file>${user.home}/.molgenis/omx/molgenis-server.properties</file>
-							</files>
-							<quiet>true</quiet>
-						</configuration>
-					</execution>
-				</executions>
+				<groupId>org.eclipse.jetty</groupId>
+				<artifactId>jetty-maven-plugin</artifactId>
+				<version>9.2.6.v20141205</version>
+				<configuration>
+					<stopPort>8079</stopPort>
+					<stopKey>jetty-stop</stopKey>
+					<reload>manual</reload>
+					<systemProperties>
+						<systemProperty>
+							<name>molgenis.home</name>
+							<value>${user.home}${file.separator}.molgenis${file.separator}omx</value>
```

Comment:
Might be a good moment to say farewell to "omx" and rename the folder to "molgenis"